scripts/model_assembly/alignment_experiment.py

```diff
@@ -7,12 +7,17 @@
 
 def main(args):
     CUR_DIR = os.getcwd()
-    MODEL_NAME = args.grfn_file.replace("--GrFN.json", "")
+    MODEL_NAME = os.path.basename(args.grfn_file).replace("--GrFN.json", "")
```

Comment:
@pauldhein, do you have any objections to getting the model name this way? otherwise, I can't pass the grfn file that is outside of the current directory.

Reply:
I'm okay with this.